_data/cryptocurrencies.yml

```
@@ -323,6 +323,15 @@ websites:
       - totp
     doc: https://support.coins.ph/hc/en-us/articles/202604924
 
+  - name: CoinSmart
+    url: https://www.coinsmart.com/
+    img: coinsmart.svg
+    tfa:
+      - totp
+    doc: https://coinspot.zendesk.com/hc/en-us/sections/115000049254
```

```suggestion
    doc: https://coinsmart.zendesk.com/hc/en-us/articles/360014948893
```
The doc link provided is for a different website called "CoinSpot". Here is the doc link I think you meant to include.